packages/jest-core/src/getConfigsOfProjectsToRun.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {Config} from '@jest/types';
+import getProjectDisplayName from './getProjectDisplayName';
+
+export default function getConfigsOfProjectsToRun(
+  argv: Config.Argv,
+  projectConfigs: Array<Config.ProjectConfig>,
+): Array<Config.ProjectConfig> {
+  if (!argv.runProjects) {
+    return projectConfigs;
+  }
+  const namesOfProjectsToRun = new Set<string>(argv.runProjects);
```

Comment:
Might make sense to warn if the name here doesn't match any project

Reply:
is this and the feedback below addressed?

Reply:
We display: "No project to run". https://github.com/yacinehmito/jest/blob/1cd4d1a28f7d9898941005779c849d09482f7109/packages/jest-core/src/getProjectsRunningMessage.ts#L15

I think it's not enough; how do you think the warning should be worded?

Suggestion: "You provided values for --selectProjects but no projects were find matching the selection".

Might be too verbose, and I don't know if the 2nd second person form is suitable.